### PR TITLE
[Testing:SubminiPolls] Add polls cypress tests

### PR DESCRIPTION
**Pull request by sbelsk**
### Please check if the PR fulfills these requirements:

* [ ] Tests for the changes have been added/updated (if possible)
* [ ] Documentation has been updated/added if relevant

### What is the current behavior?
There are no tests for polls currently, except for unit tests for the polls model and bulk export of polls (poll utils).

### What is the new behavior?
Polls now have an e2e test that runs in GitHub actions to test basic features on the instructor and student pages. 

### Other information?
The test operates on the sample course and relies on the existence of the polls that are set up automatically with the course when running `vagrant up` (implemented in https://github.com/Submitty/Submitty/pull/6780).

### CODE REVIEW

**Review by williamjallen — APPROVED** [review]

**Comment by bmcutler** [comment]
@sbelsk 
The polls tests are failing on github :(

**Comment by sbelsk** [comment]
> @sbelsk
> The polls tests are failing on github :(

@bmcutler I couldn't figure out why that test failed where the poll that gets created with the date of creation of the sample course got put into the "future polls" table. I can't say for sure what caused it, so I also can't tell if there is actually an issue with the times on the github environment vs the Submitty/DB. For now I changed the test to not get into this weird edge case

File: site/cypress/integration/polls.spec.js [hunk 1]
```diff
@@ -0,0 +1,300 @@
...
+        cy.get('#poll-date').type('1970-01-01');
+        cy.contains('+ Add Response').click();
+        cy.contains('+ Add Response').click();
+        cy.contains('+ Add Response').click();
+        cy.get('#response_0_wrapper').children(':nth-child(3)').check();
+        cy.get('#response_0_wrapper').children(':nth-child(4)').type('Answer 1');
+        cy.get('#response_1_wrapper').children(':nth-child(4)').type('Answer 2');
+        cy.get('#response_2_wrapper').children(':nth-child(3)').check();
+        cy.get('#response_2_wrapper').children(':nth-child(4)').type('Answer 3');
+        cy.get('h1').click();
```
Comment (cjreed121) [inline site/cypress/integration/polls.spec.js:164]: Could you test image upload and make sure that shows? Maybe markdown too?

**Review by cjreed121 — COMMENTED** [review]
Test looks good. Just 1 comment to improve test possibly. Will need to wait for #6865 to get merged.

**Review by cjreed121 — APPROVED** [review]

**Review by williamjallen — APPROVED** [review]

**Pull request merged**